keras_nlp/models/bert/bert_tasks_test.py

```diff
@@ -34,49 +36,79 @@ def setUp(self):
             max_sequence_length=128,
             name="encoder",
         )
-        self.classifier = BertClassifier(self.backbone, 4, name="classifier")
```

Comment:
Should we still have a test with dense inputs?

Reply:
We do I think.

Dense inputs to call -> test_valid_call_classifier
Dense inputs to predict -> test_bert_classifier_predict_no_preprocessing
Dense inptus to fit -> test_bert_classifier_fit_no_preprocessing

Something else to cover?